frontend/views/utils/translations.js

```diff
@@ -65,6 +65,15 @@ export default function L (
   return template(key, args)
 }
 
+export function LError (error) {
+  return {
+    reportError: L('Please {a_}report to us{_a} the error.', {
+      'a_': `<a class="link" target="_blank" href="/app/dashboard?modal=UserSettingsModal&section=application-logs&errorMsg=${error.message}">`,
```

Comment:
1. You must `encodeURI` the `error.message`, e.g. `encodeURI(error.message)`
2. Since we are not sure yet how this will behave when we create an app bundle out of it, please remove the `target="_blank"` part for now. You can add a comment and/or open an issue to add it back later once we are able to test it with Electron or whatever app-creator we use.

Reply:
1. Will do
2. Will we create a app for the prototype? I thought this would be browser based. I can leave a comment to review it later but for now we need the target blank

Reply:
@sandrina-p OK if you really want to keep `target="_blank"` then please open an issue referencing this problem and issue #263.

Reply:
Done. https://github.com/okTurtles/group-income-simple/issues/885